dataproc/create_cluster/create_cluster.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This sample shows how you can use the Cloud Dataproc Client library to create a
+// Cloud Dataproc cluster
+
+package dataproc
+
+// [START dataproc_create_cluster]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	dataproc "cloud.google.com/go/dataproc/apiv1"
+	"google.golang.org/api/option"
+	dataprocpb "google.golang.org/genproto/googleapis/cloud/dataproc/v1"
+)
+
+func createCluster(w io.Writer, projectID, region, clusterName string) error {
+	ctx := context.Background()
+
+	clusterClient, err := dataproc.NewClusterControllerClient(ctx,
+		option.WithEndpoint(region+"-dataproc.googleapis.com:443"),
```

Comment:
A single line is probably OK. In general, Go doesn't have a line length limit. In particular, discourages line breaks in the middle of function calls. See https://github.com/golang/go/wiki/CodeReviewComments#line-length.

However, sometimes it does make sense to have a line break to avoid having horizontal scroll when the sample is embedded in docs. In this case, it's probably short enough to have on a single line.